jobs/atc/spec

```diff
@@ -478,3 +479,11 @@ properties:
       information about the user. The only metric collected is the version of
       concourse.
     default: false
+
+  token_signing_key:
+    description: |
+      PEM RSA private key used for minting ATC tokens. If not set, one is generated
+      automatically.
+    default:
```

Comment:
I think it'd be better to omit `default` since the default isn't actually respected. You could then use `if_p` to check if it's configured.

Example: https://github.com/concourse/concourse/blob/9b041c529a54242f2ce69fae6d31ba9c0ebce31d/jobs/atc/templates/atc_ctl.erb#L96